console/cypress/integration/data/materialized-views/spec.js

```diff
@@ -0,0 +1,457 @@
+import {
+  getElementFromAlias,
+  baseUrl,
+  tableColumnTypeSelector,
+} from '../../../helpers/dataHelpers';
+
+import {
+  setMetaData,
+  validateCT,
+  createView,
+  validateColumn,
+  validateView,
+} from '../../validators/validators';
+import { setPromptValue } from '../../../helpers/common';
+
+const userId = 5555;
+
+export const Createtable = (name, dict) => {
```

Comment:
I'd stick to the camel case for functions and use `createTable` instead.

Reply:
Done!